packages/flutter/lib/src/widgets/scroll_simulation.dart

```diff
@@ -165,67 +169,146 @@ class ClampingScrollSimulation extends Simulation {
   /// The more friction the particle experiences, the sooner it stops.
   final double friction;
 
-  late double _duration;
+  late int _duration;
   late double _distance;
 
   // See DECELERATION_RATE.
   static final double _kDecelerationRate = math.log(0.78) / math.log(0.9);
 
   // See computeDeceleration().
   static double _decelerationForFriction(double friction) {
-    return friction * 61774.04968;
+    return 9.80665 *
+        39.37 *
+        friction *
+        1.0 * // Assume display density of 1.0.
```

Comment:
how much of a difference does this make? if it matters, we should be able to find a way to get the pixel density here...

Reply:
There is an API in Android but I couldn't find one in Flutter. We have `deviceRatio` which is slightly different. For what it's worth, this hasn't changed from the previous logic, just the formula is now explicit.

Reply:
although, actually, maybe 1.0 is the right number here? since we operate in logical pixels not physical pixels in the framework. In which case the comment should reflect that -- it's not an assumption, it's a reality.